addons/hr_contract/models/hr_contract.py

```diff
@@ -142,7 +142,8 @@ def _check_current_contract(self):
             if self.search_count(domain):
                 raise ValidationError(
                     _(
-                        'An employee can only have one contract at the same time. (Excluding Draft and Cancelled contracts).\n\nEmployee: %(employee_name)s',
+                        'No matter how stellar an employee might be, they\'re limited to just one contract at a time '
```

Comment:
Replace both lines by:

No matter how stellar %(employee_name)s might be, you are limited to one contract per company at a time.